.rpm-lockfiles/check-repo-access.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Quick check of repository access for Submariner RPM dependencies
+#
+# Tests if repos are accessible (subscription entitlements).
+# For full package verification, use: verify-packages.sh
+#
+# Requires: curl, entitlement certs in /etc/pki/entitlement/
+
+set -euo pipefail
+
+# Colors
+R='\033[31m' G='\033[32m' B='\033[1m' N='\033[0m'
```

Comment:
```suggestion
R=$(tput setaf 1) G=$(tput setaf 2) B=$(tput setaf 4) N=$(tput sgr0)
```

This ensures that colours are only used if the terminal supports them; in contexts where they are not desired (for example in logs), `TERM=dumb` disables them entirely (no escapes are produced).